### ISSUE TITLE
About the batch size.

### ISSUE THREAD

**Issue by yijunCai** [wanglimin/UntrimmedNet#24]
Hi, thanks for sharing the code!
I'm a little confused about the batch size. In your paper, the batch size is set to 256. However, the batch size in the prototxt is set to 7, and the actual batch size should be doubled because iter_size=2.  
(1) What exactly is the batch size used for producing the results in your paper?
(2) Let's say that the batch size is m, deos it mean that there are m videos in a batch, where each video contains 7x3 snippets?

**Comment by wanglimin** [comment]
We train UntrimmedNet with 8 GPUs, thus the batch size is: 7*2*8 = 112. To speed up the training process, we release the solver with iter_size=2. 

In our CVPR paper, we set iter_size as 4 and thus the actual batch size is 7*4*8 = 224. For proposal number = 5, we set each GPU batch size 8 and thus the batch size is 8*4*8 = 256 and for proposal number =7, we reduce each GPU batch size to 7 due to memory limit.

Batch size means the number of videos.
